src/ocean/util/container/map/HashMap.d

```diff
@@ -112,7 +112,7 @@ public class HashMap ( V ) : Map!(V, hash_t)
 
     ***************************************************************************/
 
-    public override hash_t toHash ( hash_t key )
+    public override hash_t toHash ( in hash_t key )
```

Comment:
I'd say just the hash_t overload, since they are the ones that will come as surprising to the reader.